### PR TITLE
Winbuild

### PR DESCRIPTION
**Pull request by rodwiddowson**
A handful of minor fixes to the new "winbuild" stream.  The most obvious was that I got tripped up by specifying WITH_SSL=DLL, not WITH_SSL=dll.

Another minor nit was the inability to clean from the makefile.  What I've put is slightly ropy since it still needs MODE=whatever specified but it works and is less frightening that "git clean -f -x -d".  Maybe a MODE=clean is the better solution.

### CODE REVIEW

**Comment by rodwiddowson** [comment]
Looks like Travis failed because of a previous commit...

File: lib/Makefile.vc6 [hunk 1]
```diff
@@ -51,6 +51,7 @@
 # ----------------------------------------------
 # Verify that current subdir is libcurl's 'lib'
 # ----------------------------------------------
+!MESSAGE Use of this make system is deprecated in favour of the on in winbuild.
```
Comment (BurningEnlightenment) [inline lib/Makefile.vc6]: little typo:
> of the __one__ in winbuild

File: src/Makefile.vc6 [hunk 2]
```diff
@@ -35,6 +35,7 @@
 ##
 #
 #############################################################
+!MESSAGE Use of this make system is deprecated in favour of the on in winbuild.
```
Comment (BurningEnlightenment) [inline src/Makefile.vc6]: little typo:
> of the __one__ in winbuild

**Comment by BurningEnlightenment** [comment]
@rodwiddowson As you've correctly noted the Travis build is linux only and not affected by the files you've modified, so please remove the noop commit and fix the typos.
Otherwise this looks good to me :shipit:

**Comment by rodwiddowson** [comment]
No so up with github (previous experience has been with gerrit).  Anyway, I did a rebase and a push -f.  If that was wrong I'll redo.

**Comment by bagder** [comment]
ping @mback2k and @captain-caveman2k, do I have your thumbsup for this?

**Comment by mback2k** [comment]
@bagder Yes, this PR looks good.

**Comment by captain-caveman2k** [comment]
1) I personally would do this as two changes:

a) For the tidy up stuff (as this could be added in tomorrow's release), and
b) For adding the clean option (in the next release)

It also uncouples textual type changes from functionality.

2) I wouldn't change the following type lines:

 WITH_SSL=<dll or static>     - Enable OpenSSL support, DLL or static

to

 WITH_SSL=<dll or static>     - Enable OpenSSL support, dll or static

As the part of the left indicates what the values should be and the part on the right is a description albeit  badly worded IMHO. I personally would have gone for something such as:

 WITH_SSL=<dll or static>     - Enable OpenSSL support with dynamic or static link libraries

3) Is the !MESSAGE at the top of the vc6 make files going to cause issues for the auto build error detection?

**Comment by zhangyong0538** [comment]
Thanks for your comments!

**Comment by rodwiddowson** [comment]
In response this morning's mail. 

Please do not close this.  I need to loop back around to this, but I have two or three releases to get sorted (depending on what the next curl release looks like).  

Alternatively close it and I'll just re-push it (probably next year tho - see below)

Responses to @captain-caveman2k 

> 1) I personally would do this as two changes:
I read this comment as a suggestion to the mergers, not to me so I'm not going to change my PR..  

AIUI Github allows you to do this as a merge and I feel unclean when  I don't commit-early commit-often - particularly in a review-chain.

>2) I wouldn't change the following type lines:
>
>WITH_SSL= - Enable OpenSSL support, DLL or static
>
>to
>
>WITH_SSL= - Enable OpenSSL support, dll or static

I need to review the batch file, but my memory is that "DLL" and "static" are strings that are compared in a case sensitive manner.  In that case I'll put them in quotes.  In fact I'll put them in quotes anyway.  Before someone else says it:  AIUI as a TLA DLL should be capitalized (at least in British English).  But I absolutely don't care about that.

> 3) Is the !MESSAGE at the top of the vc6 make files going to cause issues
> for the auto build error detection?

1. No, this is a deprecated build there should be little reason for an auto-build
2. again AIUI there is no autobuild in the Win32 side. so No.

Finally since everyone who is likely to care about WIN32 is listening here: 

This was to be the first of a few PR's aimed at aligning the winbuild with our (Shibboleth and thus Santuario) use of it.  I can use what is currently there with some careful abuse of NMAKE and some down stream changes in our build and so I was going to wait to the new year when we move our tool chain forward.

But it is ugly and it would be nice to make it less so.  I was saddened but not surprised at the complete lack of response to my query for [suggestions ](https://curl.haxx.se/mail/lib-2016-08/0104.html) so I guess that there is little community interest in windows.  So I feel little motivation to waste my time and my funder's money on it.  Lets apply our resources where we can.  OTOH if you guys do care, let me know and I'll push through the changes I suggested.

**Comment by rodwiddowson** [comment]
Closing this has been superseded by #1201

**Pull request closed without merging**